sdks/python/scripts/run_tox.sh

```diff
@@ -24,9 +24,10 @@
 
 ###########################################################################
 # Usage check.
-if [[ $# != 1 ]]; then
-  printf "Usage: \n$> ./scripts/run_tox.sh <tox_environment>"
+if [[ $# < 1 || $# > 2 ]]; then
+  printf "Usage: \n$> ./scripts/run_tox.sh <tox_environment> <sdk_location>"
```

Comment:
Nit: consider adding [ ] to optional argument in usage string. `"Usage: \n$> ./scripts/run_tox.sh <tox_environment> [<sdk_location>]"`

Reply:
sg.